include/dxc/DXIL/DxilConstants.h

```diff
@@ -29,7 +29,7 @@ namespace DXIL {
 const unsigned kDxilMajor = 1;
 /* <py::lines('VALRULE-TEXT')>hctdb_instrhelp.get_dxil_version_minor()</py>*/
 // VALRULE-TEXT:BEGIN
-const unsigned kDxilMinor = 8;
+const unsigned kDxilMinor = 9;
```

Comment:
Oh, I just recalled that there may be other things gated on SM 6.9 that need to be moved to a new version to prevent them from being enabled in this experiment.

One is the general RDAT for regular shader entry points in libraries.  I'm not sure what other things off the top of my head, but that's probably not the only one.

I also realize that separating PRs between the one that just bumps the version and the rest would be really helpful.

Reply:
Tex and I talked about this issue at some length. The main concern is losing the changes to accommodate the version bump.  I mean to take the commits from this PR and put them into main as the test changes facilitate 6.9, but don't require it.